Userland/Libraries/LibJS/Runtime/Temporal/Instant.cpp

```diff
@@ -191,6 +191,18 @@ BigInt* add_instant(GlobalObject& global_object, BigInt const& epoch_nanoseconds
     return result;
 }
 
+// 8.5.7 DifferenceInstant ( ns1, ns2, roundingIncrement, smallestUnit, roundingMode ), https://tc39.es/proposal-temporal/#sec-temporal-differenceinstant
+BigInt* difference_instant(GlobalObject& global_object, BigInt const& nanoseconds1, BigInt const& nanoseconds2, u64 rounding_increment, String const& smallest_unit, String const& rounding_mode)
```

Comment:
Please make `smallest_unit` and `rounding_mode` StringViews and slap a FIXME on `round_temporal_instant` to do the same

Reply:
I just fixed it already instead :)